### ISSUE TITLE
[Question] Are the older version inside /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle required if I'm only using the latest functions version? (4.x)

### ISSUE THREAD

**Issue by kemmis** [Azure/azure-functions-docker#880]
If I'm using Microsoft.NET.Sdk.Functions 4.x, are the 3.x and 2.x FuncExtensionBundles needed in my image? Or can I delete them? They take up 429mb uncompressed and trimming them out would shrink the image size significantly.

**Comment by vijaykumar911** [comment]
Hi @kemmis, will check and update you.

**Comment by vijaykumar911** [comment]
If you are using Microsoft.NET.Sdk.Functions 4.x, the 3.x and 2.x FuncExtensionBundles are not needed in your image and can be safely deleted. These bundles are used by older versions of the Azure Functions runtime and are not required by the latest version.
Deleting them can significantly reduce the size of your image, which can be beneficial for deployment and runtime performance.

**Comment by BatsIhor** [comment]
I have Azure function that uses latest runtime and I want to remove some of the packages from default Docker Image as they are flagged by Azure Defender as vulnerable. 
ow can I do that? apt-get remove?

**Comment by vijaykumar911** [comment]
Yes, you can use     "apt-get"     to remove the packages that are flagged as vulnerable by Azure Defender from the default Docker Image used by your Azure function

**Comment by vivekjilla** [comment]
Thanks for the question. As covered in the thread, if you're only targeting the latest (4.x) bundle, the older bundled Extension Bundle versions aren't required and can be trimmed to reduce image size. Closing as answered — if you hit issues after trimming on a current supported image, please open a new issue.
